features/steps/ontohub_steps.rb

```diff
@@ -21,3 +21,86 @@
     Then the output from "psql -d ontohub_test -U postgres -t -c 'SELECT * FROM users'" should not contain "charlie@example.com"
   }
 end
+
+# Steps belong to 'create repository' scenario
+
+Given(/^I visit the start page$/) do
+  visit('/')
+end
+
+Given(/^I am logged in$/) do
+  find('.top-bar-right a#sign_in_menu_link').click
+  within('form#login') do
+    fill_in 'username', with: 'ada'
+    fill_in 'password', with: 'changeme'
+    click_button 'Sign in'
+  end
+  find('.top-bar-right').find('a#user_menu_link').click
+  expect(page).to have_content('Signed in as')
+end
+
+When(/^I visit the repository creation page$/) do
+  find('.top-bar-right a#new_menu_link').click
+  click_link('New repository')
+  expect(page).to have_content('Create a new repository')
+end
+
+When(/^I create a repository$/) do
+  $repository_name = Faker::Name.title
+  within('fieldset#repository_new_name') do
+    fill_in 'Name', with: $repository_name
+  end
+  within('fieldset#repository_new_description') do
+    fill_in 'Description', with: Faker::Lorem.sentence
+  end
+  within('fieldset#repository_new_content_type') do
+    choose('Model')
+  end
+  within('fieldset#repository_new_access') do
+    choose('Public')
+  end
+  click_button 'Save'
+  expect(page).to have_content("ada / #{$repository_name}")
```

Comment:
I think the expect line should not be part of this step because you only visit a page and do not describe a check like this.